### ISSUE TITLE
Slide transitions are distracting and should be disabled

### ISSUE THREAD

**Issue by markroepke** [uc-python/intro-python-datasci#18]
RISE slides currently transition from left-to-right. This is distracting and misleading at times. Slide transitions should be disabled.

Slide transitions can be disabled by including the following in the notebook metadata:

```
{
  ...
  "rise": {
    ...,
    "transition": "none"
  }
}
```

**Comment by markroepke** [comment]
The following set of notebooks must be converted:

- [x] Introduction
- [x] Python and Jupyter Overview
- [x] Fundamentals
- [x] Packages, Modules, Methods, Functions
- [x] Importing Data
- [x] Selecting and Filtering Data
- [x] Working with Columns
- [x] Review
- [x] Summarizing Data
- [x] Summarizing Grouped Data
- [x] Joining Data
- [x] Exporting Data
- [x] Visualizing Data

**Comment by eswan18** [comment]
Final notebook fixed in 58aa94a1
